huaweicloud/services/acceptance/cdn/data_source_huaweicloud_cdn_domain_certificates_test.go

```diff
@@ -0,0 +1,64 @@
+package cdn
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func TestAccDatasourceCDNDomainCertificates_basic(t *testing.T) {
+	rName := "data.huaweicloud_cdn_domain_certificates.test"
+	enterpriseProjectIdTestName := "data.huaweicloud_cdn_domain_certificates.enterprise_project_id_filter"
+	dc := acceptance.InitDataSourceCheck(rName)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
```

Comment:
This test case needs to create data before it can be executed.
It is recommended to add a preCheck switch to avoid errors when the use case is executed randomly.